include/boost/system/boost_system.natvis

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+
+    <Type Name="boost::source_location">
```

Comment:
This should probably go into Boost.Assert as the type is defined there.

Reply:
Well, I figured I would avoid creating one more INTERFACE project with sources added, but yes it can go there.

Reply:
I moved it to Boost.Assert